index.js

```diff
@@ -1,11 +1,33 @@
-function scuberGreetingForFeet(){
-  // Write your code here!
+function scuberGreetingForFeet(rideDistance){
+  let scuberGreeting
+  if(rideDistance <= 400){
+    scuberGreeting = 'This one is on me!'
+  }else if(rideDistance > 400 && rideDistance < 2000){
+    scuberGreeting ='That will be twenty bucks.'
+  }else if(rideDistance < 2500 && rideDistance > 2000){
+    scuberGreeting = 'I will gladly take your thirty bucks.'
+  }else if (rideDistance > 2500){
+    scuberGreeting = 'No can do.'
+  }
+  return scuberGreeting
```

Comment:
I know it seems trivial, but consistent spacing that follows the "standards" is important for readability.
```suggestion
function scuberGreetingForFeet(rideDistance){
  let scuberGreeting
  if (rideDistance <= 400) {
    scuberGreeting = 'This one is on me!'
  } else if (rideDistance > 400 && rideDistance < 2000) {
    scuberGreeting ='That will be twenty bucks.'
  } else if (rideDistance < 2500 && rideDistance > 2000) {
    scuberGreeting = 'I will gladly take your thirty bucks.'
  } else if (rideDistance > 2500) {
    scuberGreeting = 'No can do.'
  }
  return scuberGreeting
```